draft-ietf-quic-manageability.md

```diff
@@ -156,7 +152,9 @@ The following information is exposed in QUIC packet headers:
 
 - "fixed bit": the second most significant bit of the first octet most QUIC
   packets of the current version is currently set to 1, for demultiplexing
-  with other UDP-encapsulated protocols.
+  with other UDP-encapsulated protocols. Endpoints that do not need this
+  demultiplexing function might assign arbitrary values. See
+  {{I-D.thomson-quic-bit-grease}} for more details.
```

Comment:
Not sure if this is completely wise before it is adopted.  Of course, the statement will remain true nonetheless, even if that is never standardized.

Reply:
Yes, I agree that we should not cite the draft here. However, we could say that the network cannot rely on this bit as the endpoint might decide to not support it (if not needed).